packages/mjml-migrate/src/migrate.js

```diff
@@ -1,4 +1,5 @@
-import fs from 'fs'
+// import fs from 'fs'
```

Comment:
You can remove it then 👍

Reply:
i will do it but about travis error i see ? It's not update or something with the code we push ? Because some errors i see eslint should not pass on it

Reply:
Might be a version conflict on your side, try to clean anything related to eslint in global ?